{"classFile":"ExceptionUtilsTest","className":"ExceptionUtilsTest","packages":"org.apache.commons.lang3.exception","methodInfos":[{"md5":"auNxMAtQQ4nlgHGc1a71KA==","methodName":"testPrintRootCauseStackTrace_ThrowableStream","parameters":"[]"},{"md5":"GJ04bL8yU7UGQ7tLi5/tIQ==","methodName":"testPrintRootCauseStackTrace_ThrowableWriter","parameters":"[]"}],"addLines":[396,422],"delLines":[396,422],"type":"MODIFY","path":"src/test/java/org/apache/commons/lang3/exception/ExceptionUtilsTest.java","pro_Commit":"ff6b0ada72106d67aef1ee21d51bacac2f3af7ac","pro_Date":"2017-04-20 03:07:05","sha1":"af8b847b3dfbb4790a794bdd0982dac8b022dab4","file_Date":"2017-04-23 04:25:52","projectName":"commons-lang","commitID":"a060733e63a9cc476342ca190d04ad027a241d33"}